price-feeder/oracle/oracle_test.go

```diff
@@ -121,7 +130,57 @@ func (ots *OracleTestSuite) TestPrices() {
 
 	prices := ots.oracle.GetPrices()
 	ots.Require().Len(prices, 1)
-	ots.Require().Equal(prices["UMEE"], sdk.MustNewDecFromStr("3.710916056220858266"))
+	ots.Require().Equal(sdk.MustNewDecFromStr("3.710916056220858266"), prices["UMEE"])
```

Comment:
Fixed order to be (expected, actual) instead of (actual, expected)

Reply:
good catch!